extension/experiments/inspectedNode/api.js

```diff
@@ -83,6 +91,10 @@ this.inspectedNode = class extends ExtensionAPI {
     return {
       experiments: {
         inspectedNode: {
+          async getNodeName(clientId) {
+            return _getNodeName(clientId);
```

Comment:
Or you could fix the issue I reported before here. By not only returning the node name, but also a boolean that says if the element is custom or not.

Reply:
Likewise, will address.